schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/datacite/view.xsl

```diff
@@ -353,9 +353,18 @@
           <!--
           Expect the entry point to be CI_Organisation
           The full name of the creator. -->
-          <datacite:creatorName nameType="Personal">
-            <xsl:value-of select="cit:party/*/cit:individual/*/cit:name/*/text()"/>
-          </datacite:creatorName>
+          <xsl:choose>
+            <xsl:when test="cit:party/*/cit:individual/*/cit:name/*/text() != ''">
+              <datacite:creatorName nameType="Personal">
+                <xsl:value-of select="cit:party/*/cit:individual/*/cit:name/*/text()"/>
+              </datacite:creatorName>
+            </xsl:when>
+            <xsl:otherwise>
+              <datacite:creatorName nameType="Organizational">
+                <xsl:value-of select="cit:party/*/cit:name"/>
```

Comment:
If the organisation is multilingual, it seems that will put all the values as text. Should not be added in separate elements with the `xml:lang` attribute?

Reply:
I think I tried and I was having XSD validation error. Will double check.

Reply:
```
Error:(26, 70) cvc-complex-type.3.2.2: Attribute 'xml:lang' is not allowed to appear in element 'datacite:creatorName'.
```
but with the latest XSD https://schema.datacite.org/meta/kernel-4.4/ this has changed.

![image](https://user-images.githubusercontent.com/1701393/206123517-c51b76b9-20d6-48a6-b796-652cdc4e48e3.png)

I propose to only put the main language value and we can update to latest XSD later.

Reply:
Sounds good.